### DISCUSSION TITLE
Does a CLI exist?

### DISCUSSION THREAD

**Discussion by beedell-roke** [padloc/padloc#700] [Q&A]
(Similarly to https://github.com/padloc/padloc/discussions/682#discussion-4850289) does a CLI exist? Bitwarden provides [`bw`](https://bitwarden.com/help/cli/).

**Comment by MaKleSoft** [comment]
Hi there! No, we don't have a CLI as of yet. I don't want to rule out the possibility for the future but right now we don't have any concrete plans to build one. Of course if you're interested in building one yourself I'd be more than happy to give you some pointers!
